CONTRIBUTING.md

```diff
@@ -9,7 +9,7 @@
     Only use what is absolutely needed.  
 
 3. Try to stay in the stock containers.  
-    If you need something in a container PR it to [my image repo](https://github.com/parkervcp/images) where I can review and pull up to the main repo.
+    If you need something in a container PR it to [my yolks repo](https://github.com/parkervcp/yolks) where I can review and pull up to the main repo.
```

Comment:
```suggestion
    If you need something in a container, open a PR in [my yolks repo](https://github.com/parkervcp/yolks) where I can review and pull it up to the main repo. In addition, there is a larger quantity of [images here](https://github.com/parkervcp/images) for you to use that have not been migrated to Yolks yet.
```

Something along these lines would provide options for both

Reply:
The suggestion has been commited